assets/scss/6-components/cta-block/_cta-block.scss

```diff
@@ -11,55 +12,43 @@ $cta-block-min-col: px-to-rem(130px);
 $cta-block-btn-height: px-to-rem(29px);
 
 .c-cta-block {
-  border: 1px solid $color-gray-light;
-  border-radius: $border-radius-b;
-
-  &--no-border {
-    border-width: 0;
-    border-radius: 0;
-  }
+  @include row-gap($size-s);
+  display: grid;
+  grid-template-columns: 1fr;
 
   &__btns {
     @include gap($size-xxs);
     display: grid;
     grid-auto-rows: $cta-block-btn-height;
-    margin-top: $size-s;
 
     // four buttons, all in a row
     &--4 {
       grid-template-columns: repeat(4, 1fr);
-
-      // buttons should become 2x2 on wide screens
-      &-stack-from-m {
-        grid-template-columns: repeat(4, 1fr);
-        @include mq($from: bp-m) {
-          grid-template-columns: repeat(2, 1fr);
-          margin-top: 0;
-        }
-      }
     }
 
     // two buttons, side by side
     &--2 {
       grid-template-columns: repeat(2, 1fr);
+    }
 
+    @include mq($from: bp-m) {
       // buttons should become 1x1 on wide screens
-      &-stack-from-m {
+      &--1-from-bp-m {
```

Comment:
What about changing this to `--1x1-from-bp-m`? As is, it makes me think there's only one button when there are two.

Reply:
Yeah I do see your thinking there. I almost called it that. But then I was thinking about how our other names not wrapped in a media query were --2 for 2 across and --4 for 4 across so I took --1 to mean 1 across. The number of subsequent rows is just a result of the number of child elements. I figured since all the CSS is saying is that each child el should span the whole row, technically it could be 1x1, 1x1x1 and so on depending on the children.

Reply:
If you still want me to change it, it's no prob! But does that line of thinking make sense to you? Or does the `--1` still throw you off? @AndrewGibson27

Reply:
Nah that's fine, go right ahead as is @ashley-hebler.